sql/deleteSql.sql

```diff
@@ -0,0 +1,60 @@
+-- select customer images for deletion
```

Comment:
Is this file a duplicate of `deleteSql.sql`?

Reply:
apologies, must have slipped through! I've removed one